.github/workflows/beam_Java_JMH.yml

```diff
@@ -19,7 +19,7 @@ on:
   schedule:
     - cron: '0 0 * * 0'
   pull_request_target:
-    paths: ['release/trigger_all_tests.json']
+    paths: ['release/trigger_all_tests.json', 'beam_Java_JMH.json']
```

Comment:
Can we put these in a subfolder? Maybe `.github/trigger_files`? That way if they accidentally get checked in it doesn't cause issues.

Reply:
Our guidance could even be to check them in, it does no harm.

Reply:
added prefix .github/trigger_files